interface/reports/collections_report.php

```diff
@@ -762,33 +764,30 @@ function checkAll(checked) {
         if (! $duncount) {
             for ($i = 1; $i <= 3; ++$i) {
                 $tmp = SLEOB::arGetPayerID($patient_id, $svcdate, $i);
-
                 if (empty($tmp)) {
-                    break;
+                    if (empty($tmp)) break;
```

Comment:
hi @juggernautsei , 
Just clarifying the code here, which seems a bit odd:
```
if (empty($tmp)) {
    if (empty($tmp)) break;
    $payerids[] = $tmp;
}
```

Reply:
Yes, I just read the note from @stephenwaite  to put the break back in the code.

Reply:
here's the original code, https://github.com/openemr/openemr/blob/master/interface/reports/collections_report.php#L763-L771